README.md

```diff
@@ -0,0 +1,224 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🖺 Entity Relationship Diagram](#er-diagram)
+  - [📽️ Project Documentation](#pro-doc)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Carpe Diem <a name="about-project"></a>
+![blog-gif](https://thumbs.gfycat.com/WarmheartedPaleHamadryad-max-1mb.gif)
+
+**Carpe Diem** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.The Ruby on Rails application that allows the user to:
+
+ - register and log in, so that the data is private to them.
+ - introduce new transactions associated with a category.
+ - see the money spent on each category.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+</details>
+
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
+
+<details>
+  <summary>Tailwind CSS / Material Tailwind </summary>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Authentication, User can manage his categories and transactions ]**
+- **[User must be authenticated]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Documentation -->
+## 📽️ Project Doocumentation <a name="pro-doc"></a>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [My-Exp](https://my-expenses-3hoz.onrender.com) 
+
+- [video] (https://www.loom.com/share/d12a806d77a647d7a6252a596f3a1251)
```

Comment:
- [ ] as the previous CR already mentioned your video is not available, please go to your video in Loom and click on share in the navbar, you should see below pop-up please make sure that the option in the red box is set to "anyone with a link can view."
![afbeelding](https://user-images.githubusercontent.com/39240820/234209376-2770d942-311a-4c6c-a8a3-e2b1a5c9a715.png)